src/sections/Advice/index.jsx

```diff
@@ -1,9 +1,64 @@
+import { useState } from "react"
+import { useEffect } from "react"
+import AdviceSlip from "./components/AdviceSlip"
+import FavouriteSlipsList from "./components/FavouriteSlipsList"
+
 function AdviceSection() {
+  const [random, setRandom] = useState(null)
+  const [firstFavourite, setFirstFavourite] = useState(null)
+  const [secondFavourite, setSecondFavourite] = useState(null)
+  const [thirdFavourite, setThirdFavourite] = useState(null)
+  const [newAdvice, setNewAdvice] = useState(null)
+  const [saveRandom, setSaveRandom] = useState(null)
+
+  useEffect(() => {
+    fetch('https://api.adviceslip.com/advice')
+      .then(res => res.json())
+      .then(setRandom)
+
+    fetch('https://api.adviceslip.com/advice/1')
+      .then(res => res.json())
+      .then(setFirstFavourite)
+      
+    fetch('https://api.adviceslip.com/advice/2')
+      .then(res => res.json())
+      .then(setSecondFavourite)
+
+    fetch('https://api.adviceslip.com/advice/3')
+      .then(res => res.json())
+      .then(setThirdFavourite)
+  }, [])
+
+  const handleClick = () => {
+    fetch('https://api.adviceslip.com/advice/4')
```

Comment:
Nice! You could save the current advice id in a state so every time you click you get a new one :)